tests/suites/test_suite_pkcs7.data

```diff
@@ -62,13 +62,13 @@ PKCS7 Signed Data Parse Failure Corrupt signerInfo.serial #15.2
 depends_on:MBEDTLS_SHA256_C
 pkcs7_parse:"data_files/pkcs7_signerInfo_serial_invalid_size.der":MBEDTLS_ERR_PKCS7_INVALID_SIGNER_INFO
 
-pkcs7_get_signers_info_set error handling (6213931373035520)
-depends_on:MBEDTLS_RIPEMD160_C
-pkcs7_parse:"data_files/pkcs7_get_signers_info_set-missing_free-fuzz_pkcs7-6213931373035520.der":MBEDTLS_ERR_PKCS7_INVALID_SIGNER_INFO + MBEDTLS_ERR_ASN1_UNEXPECTED_TAG
+PKCS7 Signed Data Parse Fail Corrupt signerInfos[2] (6213931373035520)
```

Comment:
Is this the order number?
I just saw a series of numbers from the beginning, #1, #2, #3... till #13,  then skipped #14 and to #15.1, #15.2, and then this number.  So what is't the special meaning of it?

Reply:
They used to be sequence numbers from when the PKCS7 support was originally merged - I considered removing them, but due to the truncation of test result strings, some of the tests would be indistinguishable except by their position.